using System;
using System.IO;

namespace _07_exception_class
{
    internal class Program
    {
        static void Main(string[] args)
        {
            /*int hiddenNumber = 5;
            int input;
            Console.WriteLine("Guess a number");
            input = Convert.ToInt32(Console.ReadLine());

            while (input != hiddenNumber)
            {
                Console.WriteLine("wrong guess,try again!");
                input = Convert.ToInt32(Console.ReadLine());
            }
            Console.WriteLine("correct! "+hiddenNumber);
            */
            /*int hiddenNumber = 5;
            int input;
            Console.WriteLine("gess number");
            int answer = Console.ReadLine();
            if (Int32.TryParse(answer,out guess))
            {

            }*/

           /*
                Console.WriteLine("Enter a number of lines: ");
                string answer = Console.ReadLine();
                int nr;
                if (Int32.TryParse(answer, out nr))
                {
                    try
                    {
                        StreamReader file = File.OpenText("englih.txt");
                        for (int i = 0; i > nr; i++)
                        {
                            Console.WriteLine(file.ReadLine());
                        }
                    }
                    catch
                    {
                        Console.WriteLine("Oopsie... file not found!");
                    }
                }
                else Console.WriteLine("Wrong number!");*/



            }
        }
    }
}
